aioesphomeapi/_frame_helper/noise.py

```diff
@@ -218,7 +218,7 @@ def _send_hello_handshake(self) -> None:
         frame_len = len(handshake_frame) + 1
         header = bytes((0x01, (frame_len >> 8) & 0xFF, frame_len & 0xFF))
         self._write_bytes(
-            b"".join((NOISE_HELLO, header, b"\x00", handshake_frame)),
+            [NOISE_HELLO, header, b"\x00", handshake_frame],
```

Comment:
```suggestion
            (NOISE_HELLO, header, b"\x00", handshake_frame),
```

Tuple will be slightly faster